unified-runtime/test/conformance/lit.cfg.py

```diff
@@ -7,5 +7,50 @@
 
 """
 
-# Disable lit testing (for now)
-config.suffixes = []
+import lit.formats
+import subprocess
+from os import path
+
+config.name = "Unified Runtime Conformance"
+config.test_format = lit.formats.GoogleTest("", "-test")
+config.test_source_root = config.binary_dir
+config.test_exec_root = config.binary_dir
+
+config.environment["UR_LOADER_USE_LEVEL_ZERO_V2"] = "1" if config.using_l0_v2 else "0"
+
+if "UR_CTS_ALSO_RUN_KNOWN_FAILURES" in os.environ:
+    config.environment["UR_CTS_ALSO_RUN_KNOWN_FAILURES"] = os.environ[
+        "UR_CTS_ALSO_RUN_KNOWN_FAILURES"
+    ]
+
+if "selector" in lit_config.params:
+    # Specified via passing `-Dselector="backend:*"` to lit
+    selector = lit_config.params["selector"]
+elif os.environ.get("ONEAPI_DEVICE_SELECTOR", ""):
```

Comment:
You don't need the empty string, and the double lookups are weird.
If you *really* insist on the spread out logic with multiple if statements rather than chained default, you can avoid the double lookups with assignment expressions

```python
if s := os.environ.get("ONEAPI_DEVICE_SELECTOR"):
   selector = s
 etc
```